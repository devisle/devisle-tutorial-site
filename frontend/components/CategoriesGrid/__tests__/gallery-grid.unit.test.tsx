import 'jest-styled-components';
import { render } from '../../../test/test-utils';
import GalleryGrid from '../index';

test('renders', () => {
    const { container } = render(<GalleryGrid />);
    expect(container).toMatchSnapshot();
});
