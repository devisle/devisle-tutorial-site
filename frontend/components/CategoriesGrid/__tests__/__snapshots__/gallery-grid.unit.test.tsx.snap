// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
.c0 {
  display: grid;
  grid-template-columns: repeat(,1fr);
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  grid-row-gap: 30px;
  grid-column-gap: 10px;
  margin-top: 0;
  grid-template-columns: repeat(auto-fill,minmax(200px,1fr));
}

<div>
  <section
    class="gridstyles__Grid-sc-1kg0xgl-0 c0"
  >
    <img
      alt="javascript icon"
      src="/assets/categories/js.jpg"
      style="width: 100%;"
      title="javascript"
    />
    <img
      alt="python"
      src="/assets/categories/python.png"
      style="width: 100%;"
      title="python"
    />
    <img
      alt="javascript icon"
      src="/assets/categories/js.jpg"
      style="width: 100%;"
      title="javascript"
    />
    <img
      alt="python"
      src="/assets/categories/python.png"
      style="width: 100%;"
      title="python"
    />
    <img
      alt="javascript icon"
      src="/assets/categories/js.jpg"
      style="width: 100%;"
      title="javascript"
    />
    <img
      alt="python"
      src="/assets/categories/python.png"
      style="width: 100%;"
      title="python"
    />
    <img
      alt="javascript icon"
      src="/assets/categories/js.jpg"
      style="width: 100%;"
      title="javascript"
    />
    <img
      alt="python"
      src="/assets/categories/python.png"
      style="width: 100%;"
      title="python"
    />
  </section>
</div>
`;
